MapboxNavigation/NavigationMapView.swift

```diff
@@ -1030,14 +1030,25 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         let line = MGLPolyline(coordinates: slicedLine, count: UInt(slicedLine.count))
         
         tracksUserCourse = false
+        
+        // If the user has a short distance left on the route, prevent the camera from zooming all the way.
+        // `MGLMapView.setVisibleCoordinateBounds(:edgePadding:animated:)` will go beyond what is convient for the driver.
```

Comment:
>  will go beyond what is convient for the driver.

_did you mean to say_ `convenient to the driver`?

Reply:
Yeah, it'd be inconvenient for the map to continue zooming past what is comprehensible.

Reply:
@bsudekum oh okay. There's a typo `convient for` should `convenient to`